README.md

```diff
@@ -7,10 +7,11 @@ This Git repository is a central repository for examples using official immudb c
 Examples (Python, Go, CLI) are tested against [immudb:latest](https://hub.docker.com/r/codenotary/immudb) container.
 
 More documentation at:
-- [Go](https://docs.immudb.io/master/) 
```

Comment:
Something wrong with formatting here - after the change the render is broken

Reply:
updated, it did not have to do with the .net sample anyways.